<launch>
	<arg name ="gps_port"     default = "/dev/ttyUSB0"/>
	
	 <node name="gps_node" pkg="driverless" type="gps_node" output="screen">
	   	 <param name="gps_port_name" type="string"  value = "$(arg gps_port)" />
	 </node>
	 <node name="record" pkg ="test" type ="temp_sub_node"/>
</launch>
